requirements.txt

```diff
@@ -3,7 +3,7 @@ Brotli==1.0.9
 cfgv==3.3.1
 click==8.0.3
 distlib==0.3.4
-filelock==3.4.2
+filelock>=3.4.1
```

Comment:
Why the downgrade?

Reply:
It looks like the file got downgraded  during the branch merge. I will fix it